examples/kft-v2/trl-gpt2-checkpointing.ipynb

```
@@ -0,0 +1,1215 @@
+   "source": [
+    "from kubeflow.trainer import CustomTrainer, Initializer\n",
+    "\n",
+    "training_env_args = {\n",
+    "    \"PYTHONUNBUFFERED\": \"1\",\n",
+    "    \"NCCL_DEBUG\": \"INFO\",\n",
+    "    \"TORCH_DISTRIBUTED_DEBUG\": \"INFO\",\n",
+    "    \"PYTHONPATH\": \"/tmp/lib:$PYTHONPATH\",\n",
+    "\n",
+    "    # Training hyperparameters (matching trl-trainjob.yaml)\n",
+    "    \"LEARNING_RATE\": \"5e-5\",\n",
+    "    \"BATCH_SIZE\": \"1\",\n",
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "token"

(S105)